.husky/pre-commit

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
+yarn lint-staged --debug
```

Comment:
Revert changes to this file?

Reply:
I still don't know why, but my linter doesn't work without that :(
Maybe as a workaround I can create a commit with this, then I add another commit removing it

Reply:
I removed it 👍